platform/stateful-application/ansible/roles/wordpress/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
+
+# Use the new container infrastructure
+sudo: false
+
+# Install ansible
+addons:
+  apt:
+    packages:
+    - python-pip
+
+install:
+  # Install ansible
+  - pip install ansible
+
+  # Check ansible version
+  - ansible --version
+
+  # Create ansible.cfg with correct roles_path
+  - printf '[defaults]\nroles_path=../' >ansible.cfg
+
+script:
+  # Basic role syntax check
+  - ansible-playbook tests/test.yml -i tests/inventory --syntax-check
+
+notifications:
+  webhooks: https://galaxy.ansible.com/api/v1/notifications/
```

Comment:
Maybe is a stupid question, but, are we using TravisCI?

I mean, I know this config file is for TravisCI, but, what is the reason to use travis? (i don't have problem with travis, we can use travis, but i want to know the reason)

**duplicated comment**

Reply:
I created the ansible roles using ansible-galexy command line tools and It creates the scaffold of the module, including the travisCI configuration file. Just deleted both TravisCI configuration files